tests/bootstrap_project/test/project_test.dart

```diff
@@ -0,0 +1,432 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:serverpod/serverpod.dart';
+import 'package:test/test.dart';
+import 'package:path/path.dart' as path;
+
+void main() async {
+  Directory.current = path.join(Directory.current.path, '..', '..');
+  final rootPath = Directory.current.path;
+  final cliPath = path.join(rootPath, 'tools', 'serverpod_cli');
+  final tempPath = path.join(rootPath, 'temp');
+
+  final timeout = Timeout(Duration(minutes: 5));
```

Comment:
NIT(Non blocking) - Use the `@Timeout` annotation to apply the same timeout to all tests in the file.

```suggestion
@Timeout(Duration(minutes: 5))

import 'dart:convert';
import 'dart:io';

import 'package:serverpod/serverpod.dart';
import 'package:test/test.dart';
import 'package:path/path.dart' as path;

void main() async {
  Directory.current = path.join(Directory.current.path, '..', '..');
  final rootPath = Directory.current.path;
  final cliPath = path.join(rootPath, 'tools', 'serverpod_cli');
  final tempPath = path.join(rootPath, 'temp');
```